### ISSUE TITLE
Override a previously set route

### ISSUE THREAD

**Issue by alediaferia** [perwendel/spark#277]
I'm sorry in advance if this is somehow easy but I'm new to the framework.

Is it possible to override a previously set route?

**Comment by ftomassetti** [comment]
Very interesting question :)

Looking at the code eventually it seems to add the routes into a list inside the SimpleRouterMatcher and at each request it tries to match them in order, so I guess that redefining a new route for the same path would have no effect. However there is also a method named `removeRoute` in SimpleRouterMatcher which is public and it is never invoked in the code of Spark. The problem is that the SimpleRouterMatcher instance is not exposed by the Spark class and the class is final so you cannot extend it :(

Maybe it could be possible to add a method to Spark such as:
```java
void removeRoute(String path){
   simpleRouterMatcher.removeRoute(path);
}
```

**Comment by alediaferia** [comment]
Cool! Looks like coming with a patch for it may be trivial :)

**Comment by alediaferia** [comment]
Actually I would have expected that multiple invocations for setting a route would have overridden the previous one. Plus, setting the handler to `null` would have removed the route altogether. But that's probably just me :)

**Comment by alediaferia** [comment]
This is now resolved thanks to @travisspencer here: 6b76b6aaa62195cee887150b90427d97181e1292

Thank you! :)
